app/_landing_pages/insomnia.yaml

```diff
@@ -47,27 +46,18 @@ rows:
       - blocks:
         - type: card
           config:
-            title: Test APIs
-            description: Automatically test multiple requests in your collections, with no limits on the amount of collection runs that you can execute.
+            title: Test
+            description: Test APIs and run automated tests seamlessly across collections. 
             icon: /assets/icons/insomnia/checkbox-active.svg
             cta:
               text: Learn more
               url: /insomnia/test
               align: end
+      
       - blocks:
         - type: card
           config:
-            title: Debug APIs
-            description: Debug and use APIs across your collections
-            icon: /assets/icons/insomnia/bug.svg
-            cta:
-              text: Learn more
-              url: /insomnia/debug
-              align: end
-      - blocks:
-        - type: card
-          config:
-            title: Insomnia Plugin Hub
+            title: Plugins
```

Comment:
Do we want to go straight to the plugin hub, or to the plugins landing page?

Reply:
Plugins landing page I think.